dev/setup-single-cluster

```diff
@@ -12,7 +12,7 @@ source ./dev/setup-cluster-config
 ./dev/setup-k3d
 ./dev/build-fleet
 ./dev/import-images-k3d
-./dev/setup-fleet upstream shard1,shard2,shard3
+./dev/setup-fleet upstream '[{"id":"shard0","nodeSelector":{"kubernetes.io/hostname":"k3d-upstream-server-0"}},{"id":"shard1"},{"id":"shard2","nodeSelector":{"kubernetes.io/hostname":"k3d-upstream-server-2"}}]'
```

Comment:
Would this work over several lines, eg. something like this, which would be more readable?
```suggestion
./dev/setup-fleet upstream '[
    {"id":"shard0","nodeSelector":{"kubernetes.io/hostname":"k3d-upstream-server-0"}},
    {"id":"shard1"},
    {"id":"shard2","nodeSelector":{"kubernetes.io/hostname":"k3d-upstream-server-2"}}
    ]'
```

Reply:
I'll try thx